tests/Api/MauticApiTestCase.php

```diff
@@ -73,6 +145,12 @@ protected function standardTestGetListOfSpecificIds()
             $itemIds[] = $response[$this->api->itemName()]['id'];
         }
 
+        if (is_callable($callback)) {
+            // Allow support to make additional tests based on created associations with this item that
+            // may not have create/edit endpoints
+            $callback($response);
```

Comment:
Isn't this possible since PHP 5.4? Although it's probably not that important to keep PHP version for tests. It won't affect production code.